src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/DesignerOptionService.cs

```diff
@@ -48,7 +48,7 @@ protected DesignerOptionCollection CreateOptionCollection(DesignerOptionCollecti
 
             if (name.Length == 0)
             {
-                throw new ArgumentException(SR.Format(SR.InvalidArgumentValue, name.Length.ToString(), "0"), "name.Length");
+                throw new ArgumentException(SR.Format(SR.InvalidArgumentValue, "name.Length", "0"), nameof(name));
```

Comment:
Might be better to just have a string like "'{0}' cannot be empty." or similar (we have many such messages in other libraries)

Reply:
I assume it was for reusing, but seems the message only used here, so will update accordingly